pkg/api/controller.go

```diff
@@ -2406,11 +2406,7 @@ func (c *Controller) CopyObject(w http.ResponseWriter, r *http.Request, body Cop
 		return
 	}
 
-	copyType := httpHeaderCopyTypeShallow
-	if fullCopy {
-		copyType = httpHeaderCopyTypeFull
-	}
-	w.Header().Set(httpHeaderCopyType, copyType)
+	w.Header().Set(httpHeaderCopyType, httpHeaderCopyTypeFull)
```

Comment:
We don't need the header anymore + we can remove `httpHeaderCopyTypeFull` `httpHeaderCopyTypeShallow`

Reply:
Isn't it still useful for our metrics?

Reply:
I'm asking because I don't remember.

Reply:
It was added originally to provide the user information about the copy type. I don't think it is longer needed.